libra_validation/fssh_based_2_state_models/recipes.py

```diff
@@ -157,3 +160,347 @@ def make_all_sets():
+                               3rd: Representation (case) (c)
+                                   0: adiabatic init, adiabatic TDSE
+                                   1: diabatic init, adiabatic TDSE
+                                   2: adiabatic init, diabatic TDSE
+                                   3: diabatic init, diabatic TDSE
+                                   4: adiabatic init, adiabatic Liouville
+                                   5: diabatic init, adiabatic Liouville
+                                   6: adiabatic init, diabatic Liouville
+                                   7: diabatic init, diabatic Liouville
+                               4th: Ehrenfest or state-resolved options (D)
+                                   0: state-resolved (e.g. TSH) with adiabatic forces
+                                   1: for Ehrenfest in adiabatic rep
+                                   2: for Ehrenfest in diabatic rep
+                               5th: Surface hopping options (E)
+                                   0: adiabatic, no surface hopping
+                                   1: FSSH
+                                   2: GFSH
+                                   3: MSSH
+                                   4: LZ
+                                   5: ZN
+                                   6: DISH
+                               6th: Decoherence options (F)
+                                   0: no decoherence
+                                   1: mSDM
+                                   2: ID-A
+                                   3: MF-SD
+                                   4: BCSH
+                               7th: Decoherence times option (G)
+                                   0: No decoherence times, infinite decoherence times
+                                   1: EDC + default params
+                                   2: Schwartz version 1
+                                   3: Schwartz version 2
+                               8th: Hop acceptance method (H)
+                                   0: E+, accept and rescale based on total energy, do not reverse on frustrated
+                                   1: E-, accept and rescale based on total energy, reverse on frustrated
+                                   2: D+, accept and rescale based on NAC vectors, do not reverse on frustrated
+                                   3: D-, accept and rescale based on NAC vectors, reverse on frustrated
+                                   4: F+, accept and rescale based on force differences, do not reverse on frustrated
+                                   5: F-, accept and rescale based on force differences, reverse on frustrated
+                               9th: NAC update method (I)
+                                   0: Explicit
+                                   1: HST
+                                   2: NPI
+                               10th: Nuclear phase correction by Shenvi, Subotnik, and Yang (SSY) (J)

```

Comment:
i think, the next position could be given to the "integrator" - note that Zhu's Hamiltonian (for Liouville's picture is activated as one of the "integrator" options, maybe that should be revised to not confuse it with the actual numerical integration algorithm choice)

Reply:
this should be used as an alternative to the "SSY", so another possible option is to add it as option "2" in SSY. In principle, this should be a "physical phase correction" option, maybe we can replace the "do_ssy" to the "phase_correction". This is different from the numerical phase correction i discussed in my JPCL